package.json

```diff
@@ -46,7 +46,7 @@
     "css-loader": "^5.2.7",
     "eslint": "^7.32.0",
     "eslint-plugin-jest": "^24.4.2",
-    "html-webpack-plugin": "^4.5.2",
+    "html-webpack-plugin": "^5.5.0",
```

Comment:
@NikoRoberts `v5.*` only supports `webpack 5`. This will break the break documentation builder:

```
yarn build-docs
yarn run v1.22.17
$ webpack --config ./docs/webpack.config.js
[webpack-cli] TypeError: Cannot read property 'tap' of undefined
    at HtmlWebpackPlugin.apply (/Users/yuzu/code/airtasker/spot/node_modules/html-webpack-plugin/index.js:40:31)
    at WebpackCLI.webpack (/Users/yuzu/code/airtasker/spot/node_modules/webpack/lib/webpack.js:51:13)
    at WebpackCLI.createCompiler (/Users/yuzu/code/airtasker/spot/node_modules/webpack-cli/lib/webpack-cli.js:2053:29)
    at async WebpackCLI.buildCommand (/Users/yuzu/code/airtasker/spot/node_modules/webpack-cli/lib/webpack-cli.js:2173:20)
    at async Command.<anonymous> (/Users/yuzu/code/airtasker/spot/node_modules/webpack-cli/lib/webpack-cli.js:850:25)
    at async Promise.all (index 1)
    at async Command.<anonymous> (/Users/yuzu/code/airtasker/spot/node_modules/webpack-cli/lib/webpack-cli.js:1495:13)
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

An upgrade to Webpack v5 is not that simple (I tried a while ago) unfortunately.

Reply:
should this be something we add to the test suite?